ftw/contacts/configure.zcml

```diff
@@ -41,23 +35,19 @@
         />
 
     <!-- ftw.geo -->
-    <adapter
-        for="ftw.contacts.interfaces.IContact"
-        provides="collective.geo.settings.interfaces.IGeoCustomFeatureStyle"
-        factory="collective.geo.contentlocations.geostylemanager.GeoStyleManager"
-        />
+    <configure zcml:condition="installed ftw.geo">
 
-    <subscriber
-        for="ftw.contacts.interfaces.IContact
-             zope.lifecycleevent.interfaces.IObjectModifiedEvent"
-        handler=".subscribers.initializeCustomFeatureStyles"
-        />
+        <include package=".geo" />
 
-    <adapter
-        factory=".adapters.ContactLocationAdapter"
-        for="ftw.contacts.interfaces.IContact"
-        provides="ftw.geo.interfaces.IGeocodableLocation"
+        <genericsetup:registerProfile
+            name="geo"
```

Comment:
I'd probably move the complete profile into the `geo` subpackage and name it `profile-ftw.contacts.geo:default`.
By doing that we can contain everything depending on the extra into the `geo` subpackage and load it only when the extras was used (see ftw.autofeature). What do you think?